src/backend/app/auth/auth_routes.py

```diff
@@ -252,8 +263,8 @@ async def my_data(
     return await get_or_create_user(db, current_user)
 
 
-@router.get("/refresh", response_model=AuthUserWithToken)
-async def refresh_token(
+@router.get("/refresh/mgmt", response_model=AuthUserWithToken)
```

Comment:
Explicit is probably preferred over abbreviated: this could be /refresh/management 👍